File: LeetCode/[E] - 283. Move Zeroes/solution.py
class Solution:
    def moveZeroes(self, nums: list[int]) -> None:
        p_zero = 0
        for p_value in range(len(nums)):
            if nums[p_value] != 0:
                nums[p_zero], nums[p_value] = nums[p_value], nums[p_zero]
                p_zero += 1

File: LeetCode/[E] - 283. Move Zeroes/solution.go
package main

func moveZeroes(nums []int) {
	pZero := 0
	for pValue := 0; pValue < len(nums); pValue++ {
		if nums[pValue] != 0 {
			nums[pValue], nums[pZero] = nums[pZero], nums[pValue]
			pZero++
		}
	}
}
